### ISSUE TITLE
There is no detection count sockets or cpu's

### ISSUE THREAD

**Issue by GermanAizek** [hrydgard/ppsspp#18758]
### What should happen

@hrydgard, hi again.
I noticed flaw and would like to help fix it, there is not enough information about count sockets or cpu's. For example, task manager reports. It is customary to write definition processor like this as 2xE5-XXXX vN or 4xE7-XXXX vN

![image](https://github.com/hrydgard/ppsspp/assets/21138600/97641a7e-5e76-4b33-96a9-510d55eb0081)

### Who would this benefit

![299292557-43f87ef4-9abe-4d36-ac3c-62a379c6f378](https://github.com/hrydgard/ppsspp/assets/21138600/b3c1fe62-c129-4fe0-8742-6c563f684a46)

![image](https://github.com/hrydgard/ppsspp/assets/21138600/42ef6a0a-7b89-401f-a120-cedc6ff55e0d)

### Platform (if relevant)

None

### Games this would be useful in

-

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by anr2me** [comment]
Probably because it doesn't really matters how many CPU sockets are there in the motherboards that have CPU installed, most emulators only need the type of the CPU (which related to the features it supported), and how many threads it can run simultaneously.

**Comment by hrydgard** [comment]
yeah, this is not really important. use some other tool if you want accurate information, this is just meant as a sanity check that we detect about the right number of cores. The emulator doesn't care what CPUs there are exactly.
